Automattic/wp-calypso
# Update design of plan step from Sensei onboarding

## Proposed Changes

* It updates the Sensei onboarding plan design, in order to match other flows.
* It doesn't change any functionality.

## Considered Alternatives

* I started the working trying to implement it tied to the Calypso code, similar to https://github.com/Automattic/wp-calypso/pull/75174 and https://github.com/Automattic/wp-calypso/pull/76358.
* After some work, I realized that we have some different integrations that seemed a little complex to port to the mentioned approach. So, after talking with Zen to collect previous experiences, I decided to keep the current approach: try to use the available flexible components from Calypso and tweak some styles to match the other flows.
* Maybe in the future we'll want to invest more time trying to use the mentioned approach with our custom integrations. So it would use the same components as the other flows.

## Testing Instructions



* Navigate to `/setup/sensei/senseiPlan`.
* Make sure it works properly.

I asked for design and texts review here: p1684871185282869-slack-C048ZJB2QNM

## Pre-merge Checklist




## Screenshots

<img width="1486" alt="Screenshot 2023-05-23 at 15 53 54" src="https://github.com/Automattic/wp-calypso/assets/876340/df9d607c-5974-4c2a-837c-d119fde21b4e">

<img width="1484" alt="Screenshot 2023-05-23 at 15 54 03" src="https://github.com/Automattic/wp-calypso/assets/876340/ca5dfbfc-fd1d-471d-9978-95dc1fcaae0d">
